starboard/build/config/BUILDCONFIG.gn

```diff
@@ -451,6 +451,88 @@ template("shared_library") {
       }
     }
 
+    # Add an additional evergreen_loader target to every gtest target.
+    if (sb_is_evergreen_compatible && sb_evergreen_compatible_package &&
+        target_name != "loader_app") {
+      original_target_name = target_name
+      data_deps += [ "//third_party/icu:icudata" ]
+      data_deps += [
+        ":copy_loader_app_lib${original_target_name}",
+        "//starboard/loader_app:copy_loader_app_content",
+      ]
+      if (cobalt_font_package == "empty") {
+        data_deps += [ "//cobalt/content/fonts:copy_font_data" ]
+      } else {
+        data_deps += [
+          "//cobalt/content/fonts:copy_fonts",
+          "//cobalt/content/fonts:fonts_xml",
+        ]
+      }
+      shared_library(original_target_name + "_evergreen_loader") {
+        forward_variables_from(invoker, [ "testonly" ])
+        if (has_pedantic_warnings) {
+          configs += [ "//starboard/build/config:pedantic_warnings" ]
+        }
+        configs += [ toolchain_config_path ]
+        if (!has_pedantic_warnings) {
+          configs += [ "//starboard/build/config:no_pedantic_warnings" ]
+        }
+
+        sources = [ "//starboard/elf_loader/sandbox.cc" ]
+        configs += [ "//starboard/elf_loader:elf_loader_config" ]
+
+        deps = [
+          ":copy_loader_app_lib${original_target_name}",
+          "//cobalt/content/fonts:copy_font_data",
+          "//starboard",
+          "//starboard/elf_loader:constants",
+          "//starboard/elf_loader:elf_loader",
+          "//starboard/elf_loader:evergreen_info",
+          "//starboard/elf_loader:sabi_string",
+          "//starboard/loader_app:copy_loader_app_content",
+        ]
+
+        if (!sb_is_evergreen_compatible) {
+          deps += [ "//third_party/crashpad/wrapper:wrapper_stub" ]
+        }
+      }
+      if (separate_install_targets_for_bundling) {
+        bundle_content(
+            "copy_loader_app_lib${original_target_name}_bundle_content") {
+          forward_variables_from(invoker, [ "testonly" ])
+
+          bundle_name = shared_library_target_name + "_evergreen_loader"
+          bundle_deps = data_deps
+        }
+      }
+      copy("copy_loader_app_lib${original_target_name}") {
+        install_content = true
+        if (target_cpu == "arm" && arm_float_abi == "softfp") {
+          sources = [ "$root_out_dir/../evergreen-$target_cpu-${arm_float_abi}_$build_type/lib${original_target_name}.so" ]
+        } else if (target_cpu == "arm64") {
+          sources = [ "$root_out_dir/../evergreen-$target_cpu_$build_type/install/lib/libcobalt.so" ]
+        } else {
+          sources = []
```

Comment:
Does `sources = []` not fail in a `copy` target? If it does, this should be an `assert(false)` or the `copy` should be protected by conditionals